tests/php/mocks/ProductQueryMock.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Tests\Mocks;
+
+use Automattic\WooCommerce\Blocks\BlockTypes\ProductQuery;
+use Automattic\WooCommerce\Blocks\Package;
+use Automattic\WooCommerce\Blocks\Assets\Api;
+use Automattic\WooCommerce\Blocks\Assets\AssetDataRegistry;
+use Automattic\WooCommerce\Blocks\Integrations\IntegrationRegistry;
+
+/**
+ * ProductQueryMock used to test cart block functions.
+ */
+class ProductQueryMock extends ProductQuery {
+
+	/**
+	 * We initaite our mock class.
```

Comment:
```suggestion
	 * We initialize our mock class.
```

`initialize`? `instantiate`?